dev-tools/cloud/terraform/main.tf

```diff
@@ -0,0 +1,45 @@
+terraform {
+  required_version = ">= 0.12.29"
+
+  required_providers {
+    ec = {
+      source  = "elastic/ec"
+      version = "0.5.1"
+    }
+  }
+}
+
+provider "ec" {}
+
+variable "elastic_agent_docker_image" {
+  type        = string
+  description = "Elastic agent docker image with tag."
+}
+
+locals {
+  match           = regex("const DefaultVersion = \"(.*)\"", file("${path.module}/../../../version/version.go"))[0]
+  stack_version   = format("%s-SNAPSHOT", local.match)
```

Comment:
Intended to just run SNAPSHOT versions ? Would it be useful to run for stable versions too like `8.7.0` ?